uniffi/Cargo.toml

```diff
@@ -18,7 +18,7 @@ ffi-support = "~0.4.2"
 lazy_static = "1.4"
 log = "0.4"
 # Regular dependencies
-cargo_metadata = "0.12"
+cargo_metadata = "0.11"
```

Comment:
The latest `cargo_metadata` depends on a version of the `semver` crate that uses features not available in older Rust.